stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/application/App.kt

```diff
@@ -0,0 +1,25 @@
+package io.getstream.chat.ui.sample.application
+
+import android.app.Application
+import io.getstream.chat.ui.sample.data.user.UserRepository
+
+class App : Application() {
+
+    lateinit var chatInitializer: ChatInitializer
```

Comment:
Instead of `lateinit var` we can use lazy

Reply:
This is the intended use case for `lateinit`, I think it's good as-is